src/app/global/components/nav-desktop/nav-desktop.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+
+interface Item {
+  icon: string;
+  rota: string;
+  href: string;
+}
```

Comment:
esse "rota" na verdade não é o nome do item e a rota é o href?
```suggestion
interface Item {
  icon: string;
  nome: string;
  rota: string;
}
```